controllers/preview-pane.js

```diff
@@ -0,0 +1,56 @@
+const dom = require('@nymag/dom'),
+  _ = require('lodash'),
+  previewSizes = {
+    small: { w: 375, h: 660 },
+    medium: { w: 768, h: 1024 },
```

Comment:
I think our "medium" screen size in clay is 600? That's what we base our css off of at least